nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/app/pages/flow-designer/ui/manage-remote-ports/manage-remote-ports.component.ts

```diff
@@ -0,0 +1,369 @@
+                        filter((entity) => entity != null),
+                        take(1)
+                    )
+                ),
+                concatLatestFrom(() => [this.store.select(selectRpg).pipe(isDefinedAndNotNull())]),
+                takeUntilDestroyed()
+            )
+            .subscribe(([entity, rpg]) => {
+                if (entity) {
+                    this.store.dispatch(
+                        openConfigureRemotePortDialog({
+                            request: {
+                                id: entity.id,
+                                port: entity,
+                                rpg
+                            }
+                        })
+                    );
+                }
+            });
+    }
+
+    ngOnInit(): void {
+        this.store.dispatch(loadFlowConfiguration());
+        this.store.dispatch(loadAbout());
+    }
+
+    isInitialLoading(state: RemotePortsState): boolean {
+        // using the current timestamp to detect the initial load event
+        return state.loadedTimestamp == initialState.loadedTimestamp;
+    }
+
+    refreshManageRemotePortsListing(): void {
+        this.store.dispatch(
+            loadRemotePorts({
+                request: {
+                    rpgId: this.currentRpgId
+                }
+            })
+        );
+    }
+
+    formatName(entity: PortSummary): string {
+        return entity.name;
+    }
+
+    formatTasks(entity: PortSummary): string {
+        return entity.concurrentlySchedulableTaskCount ? `${entity.concurrentlySchedulableTaskCount}` : 'No value set';
+    }
+
+    formatCount(entity: PortSummary): string {
+        if (!this.isCountBlank(entity)) {
+            return `${entity.batchSettings?.count}`;
+        }
+        return 'No value set';
+    }
+
+    isCountBlank(entity: PortSummary): boolean {
+        return (
+            entity.batchSettings &&
+            (!this.nifiCommon.isDefinedAndNotNull(entity.batchSettings.count) || entity.batchSettings.count === '')
+        );
+    }
+
+    formatSize(entity: PortSummary): string {
+        if (!this.isSizeBlank(entity)) {
+            return `${entity.batchSettings?.size}`;
+        }
+        return 'No value set';
+    }
+
+    isSizeBlank(entity: PortSummary): boolean {
+        return (
+            entity.batchSettings &&
+            (!this.nifiCommon.isDefinedAndNotNull(entity.batchSettings.size) || entity.batchSettings.size === '')
+        );
+    }
+
+    formatDuration(entity: PortSummary): string {
+        if (!this.isSizeBlank(entity)) {
+            return `${entity.batchSettings?.duration}`;
+        }
+        return 'No value set';
+    }
+
+    isDurationBlank(entity: PortSummary): boolean {
+        return (
+            entity.batchSettings &&
+            (!this.nifiCommon.isDefinedAndNotNull(entity.batchSettings.duration) ||
+                entity.batchSettings.duration === '')
+        );
+    }
```

Comment:
We should be able to simplify this based on the updated type definition. I think it should just be

`return this.nifiCommon.isBlank(entity.batchSettings.duration);`